DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: gcamdata
 Type: Package
 Title: Build the GCAM Input Datasets
-Version: 1.0
+Version: 5.1
```

Comment:
I thought we had said were not going to sync version numbers? No problem either way, just wondering what changed your mind.

Reply:
Yea, I still don't have any strong feelings.  @kvcalvin thought it would be better however.

Reply:
I think it will be easier for users if numbers were aligned. Didn't realize we hadn't run it by you though. Sorry.

Reply:
👍